Riot/Categories/MXSpaceChildInfo.swift

```diff
@@ -0,0 +1,33 @@
+// 
+// Copyright 2021 New Vector Ltd
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import Foundation
+
+extension MXSpaceChildInfo {
```

Comment:
We should use RoomAvatarView or factorize this code for existing place where RoomAvatarView is not used. It seems copy paste several time.

Reply:
@SBiOSoftWhare I don't understand your comment. `setRoomAvatarImage` is a method called twice alongside the method `setRoomAvatarImage` of the `MXRoomSummary`.

Reply:
In fact if we can't use `AvatarImageView` subclass we can make an intermediate extension on MKImageView like this to avoid code duplication:

extension MXKImageView {
  func vc_setRoomAvatarImage(with url: String?, displayName: String, mediaManager: MXMediaManager) {
  }
}